bin/cli.js

```diff
@@ -45,9 +45,10 @@
  * the License.
  */
 
-let scope = {}
+let scope = new Map()
```

Comment:
TODO: use `createEmptyMap()`

Reply:
Just curious: what would `createEmptyMap()` add over just calling `new Map()`?

Reply:
I'm putting a lot of indirection here to centralize some of the scope management. This is just here for completion.